Docs/source/pages/get-started.md

```diff
@@ -486,6 +486,76 @@ AltTester® package contains AltDriver class used to connect to the instrumented
                 .. literalinclude:: ../_static/examples~/common/python-ios-test.py
                     :language: py
 
+    .. tab:: robot
+
+        **Prerequisite:**
+            - `Robot Framework <https://robotframework.org/?tab=1#getting-started>`_ installed.
+            - Instrumented build.
+            - The AltTester® Desktop installed on your PC.
+
+        **Connection:**
+            Starting with version ``2.0.0``, the AltTester® Desktop must be running on your PC while the tests are running.
+
+            1. The AltTester® Desktop app is running.
+            2. The instrumented build is running.
+            3. Configure the IP of the build in order to match the IP of the machine (PC) the Desktop app is running on.
+            4. Then press the restart button in the AltTester® dialog and wait until the `Connected to AltTester® Server…` message is displayed in the dialog.
+
+        **Test Setup:**
+            - You can use any IDE you want. As a suggestion, we prefer `Visual Studio Code <https://code.visualstudio.com/>`_  and `PyCharm <https://www.jetbrains.com/pycharm/download/>`_.
+
+        **Writing tests:**
+            - First you need to create a new project with th Robot class under it, which can store your tests
```

Comment:
There is a typo here -> "th". Maybe you can replace it with "... a Robot class under it".